libs/openFrameworks/sound/ofRtAudioSoundStream.h

```diff
@@ -5,6 +5,7 @@
 #include "ofBaseSoundStream.h"
 #include "ofTypes.h"
 #include "ofSoundBuffer.h"
+#include "RtAudio.h"
 
 class RtAudio;
```

Comment:
Removing the forward declaration `class RtAudio;` causes compile time errors with the header specified above?